templates/lxc-ubuntu.in

```diff
@@ -343,6 +343,9 @@ download_ubuntu()
     fi
     packages_template="${packages_template},$(echo $langpacks | sed 's/ /,/g')"
 
+    if [ $variant == 'minbase' ]; then
```

Comment:
unnecessary bashism, change it to:

    [ "$variant" = "minbase" ]; then

And that will work with all shells.